Makefile

```diff
@@ -20,7 +20,7 @@ test: ## Run tests
 
 .PHONY: integration-test
 integration-test: ## Run integration tests
-	python integration_test/integration_tests.py
+	python -m integration_test.integration_tests
```

Comment:
This is needed because the tests are no longer packaged with the library. Using the script form changes the python path whereas this keeps it at the root of the repo. It worked before by installing the tests into site packages and importing from there rather than the repo.

I can't properly check this though as I can't run integration tests